app/models/measurement.rb

```diff
@@ -10,11 +9,8 @@ class Measurement < ActiveRecord::Base
 
   #input: measurement
   def bmi_calc
-    #first = last, because of ordering in model
-    weight = self.weight
-    height = self.person.length
+    height = person.length
```

Comment:
You're using `height` only once in this method. Why not simple substitute `person.length` in the calculation below?